dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties

```diff
@@ -3,7 +3,6 @@ implementation=objectstorage
 # Configuration for swift BlobStore
 objectstorage.payload.codec=DEFAULT
 objectstorage.provider=swift
-objectstorage.namespace=james
```

Comment:
I'm not really into blob store things, but could it be interesting to keep a namespace (a little like a prefix), to allow an admin to configure different james on the same object store, or different overlaping applications?

Reply:
namespace here is the bucket/container. So what do you want? a configurable default bucket or a configurable bucket prefix?

I agree with both.

Reply:
I'm more thinking about a prefix

Reply:
it's a bit dangerous, where the administrators change this configuration, then restart james, then buckets were created before will be useless (data lost)

Reply:
right, but the case were 2 James (prod and preprod for example) are using the same objectstore is worrying me a little more. Maybe we should ask our dear ops what they think about it

Reply:
Yay ! Thanks for asking :)
Indeed we already deployed 2 OpenPaaS platform in the same k8s cluster, and they might use the same objectstore. So prefix in bucket name could be helpful.
Don't worry about configuration changes; If I change my cassandra or elasticsearch connection string I don't expect to keep finding my data, or I'm a dumbass.

Reply:
then it also requires to have a configurable default bucket. Otherwise prod & preprod will end up to use the same default bucket